test/NameRegistry.upgrade.t.sol

```diff
@@ -99,6 +114,12 @@ contract NameRegistryUpgradeTest is Test {
         nameRegistryProxy.upgradeTo(address(nameRegistryV2Impl));
         return NameRegistryV2(address(proxy));
     }
+
+    /// TODO: Should be moved to shared test suite for NameRegistry. (GH Issue #209)
```

Comment:
I think this can be moved into NameRegistryTestSuite and the todo can be resolved